### PR TITLE
removing unused dependencies

### PR DESCRIPTION
**Pull request by turbobeast**
* updating react-scripts
* using prop-types lib
* alt added to img tag

### CODE REVIEW

File: src/components/Card.js [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Card = ({id, name, email}) => {
   return (
     <div className='grow bg-light-green br3 pa3 ma2 dib'>
-      <img role='presentation' src={`//robohash.org/${id}?size=200x200`} />
+      <img alt={name} src={`//robohash.org/${id}?size=200x200`} />
```
Comment (rbrander) [inline src/components/Card.js:7]: Nice! Good call

File: package.json [hunk 2]
```diff
@@ -1,22 +1,19 @@
 {
-  "name": "react-training-RoboDex",
+  "name": "react-training-robodex",
```
Comment (rbrander) [inline package.json]: I think the 'react' part should be 'redux' to match the repo name

File: package.json [hunk 3]
```diff
@@ -1,22 +1,19 @@
...
   "devDependencies": {
     "gh-pages": "^0.12.0",
-    "react-scripts": "0.8.4"
+    "react-scripts": "^0.9.5"
   },
   "dependencies": {
-    "fetch": "^1.1.0",
-    "immutable": "^3.8.1",
+    "prop-types": "^15.5.7",
     "react": "^15.4.1",
```
Comment (rbrander) [inline package.json]: Since the prop-types are being used; I think we should bump the react version to 15.5.0.

File: package.json (hunk 2 above)
Comment (turbobeast) [reply package.json]: makes sense to me

Comment (turbobeast) [reply package.json]: ...actually, maybe we should change the name of this and the repo. it's not really intended to teach redux.

**Comment by rbrander** [comment]
LGTM :shipit:

**Pull request merged**